config_test.go

```diff
@@ -59,3 +59,22 @@ aliases:
 		t.Errorf("failed to parse config: %s", err)
 	}
 }
+
+func TestReviewerConfigParseComment(t *testing.T) {
+	config := types.DreckConfig{}
+	parseConfig([]byte(`reviewers:
+- aa with any comment following the github handle
```

Comment:
There should aways be a `#`, lets just make that mandatory (at least when you write a comments)

Reply:
ok